### PR TITLE
chore(sample data): Update DNs for sample data users so they work with

### PR DESCRIPTION
**Pull request by arw180**
basic auth

### CODE REVIEW

**Comment by arw180** [comment]
Fixes login for Obrien and Big Brother 2

File: ozpcenter/scripts/sample_data_generator.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def run():
         organizations=['Ministry of Peace'],
         stewarded_organizations=['Ministry of Peace', 'Ministry of Plenty'],
         groups=['ORG_STEWARD'],
-        dn='O\'Brien obrien'
+        dn='OBrien obrien'
```
Comment (clarkjefcoat) [inline ozpcenter/scripts/sample_data_generator.py]: This needs to be 'Obrien obrien' (lowercase b) to match the change in auth_data.json.

Comment (arw180) [inline ozpcenter/scripts/sample_data_generator.py]: I changed the corresponding entry in `auth_data.json` to match this DN

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**